cluster/addons/fluentd-gcp/fluentd-gcp-ds.yaml

```diff
@@ -61,14 +61,14 @@ spec:
               then
                 exit 1;
               fi;
-              LAST_MODIFIED_DATE=`stat /var/log/fluentd-buffers | grep Modify | sed -r "s/Modify: (.*)/\1/"`;
-              LAST_MODIFIED_TIMESTAMP=`date -d "$LAST_MODIFIED_DATE" +%s`;
-              if [ `date +%s` -gt `expr $LAST_MODIFIED_TIMESTAMP + $STUCK_THRESHOLD_SECONDS` ];
+              touch -d "@$(date -d"${STUCK_THRESHOLD_SECONDS} seconds ago" '+%s')" /tmp/marker-stuck;
```

Comment:
Do these files in `/tmp` need to be cleaned up?

Reply:
I'd say no, they're two empty files that are not mounted onto the host, and should get cleaned up when the container goes away anyways.